src/components/nfts/NFTCardGallery.scss

```diff
@@ -0,0 +1,91 @@
+#card-gallery-container {
+  width: 280px;
+  padding: 15px 15px 24px 15px;
+  background-color: #21383d;
+  border-radius: 5px;
+  box-sizing: border-box;
+  cursor: pointer;
+
+  p {
+    margin: 0;
+  }
+
+  header {
+    display: flex;
+    margin-bottom: 15px;
+    column-gap: 8px;
+    justify-content: center;
+
+    p {
+      font-size: 14px;
+      font-weight: 700;
+      color: #88a2b3;
+    }
+  }
+
+  main {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
```

Comment:
Yumi has created some global styles into [app.scss](https://github.com/CityOfZion/dora/blob/04d7fda1c1a283770b282dabdb736457c7a45fac/src/App.scss#L41) .
Please, check if it's possible to reuse it